File: sem1/js/12.grade.js
var regno = 20130132;
var name = "Vishnu Suresh";
var mark = 72;
switch(Math.floor(mark/10)){
    case '10':
        grade = "S";
        break;
    case '9':
        grade = "S"
        break;
    case 8:
        grade = "A";
        break;
    case 7:
        grade = "B"
        break;
    case 6:
        grade = "C";
        break;
    case 5:
        grade = "D"
        break;
    case 4:
        grade = "E";
        break;
    default:
        grade = "Failed"
        break;
}
console.log("regno : "+regno)
console.log("name : "+name)
console.log("mark : ",mark)
console.log("grade : "+grade)

File: 1008_computing_fundamentals/12.grade.py
regno = input("enter regno of the student\n")
name = input("enter name of the student\n")
mark = int(input("enter mark of the student\n"))
def dict(mark):
    switcher={
        10:"S",
        9:"S",
        8:"A",
        7:"B",
        6:"C",
        5:"D",
        4:"E",
    }
    return switcher.get(mark,"Failed")
print("regno : "+regno)
print("name : "+name)
print("mark : ",mark)
print("mark//10 : ",mark//10)
print("grade : "+dict(mark//10))
